learn/get-started/get-started.md

```diff
@@ -0,0 +1,138 @@
+
+`generate_keypair.js` file:
+```js
+var StellarSdk = require('stellar-sdk');
+var keypair = StellarSdk.Keypair.random();
+
+console.log('Account ID:');
+console.log(keypair.accountId());
+console.log('Secret Seed:');
+console.log(keypair.seed());
+```
+
+Now, let's execute the file:
+```
+$ node generate_keypair.js
+
+Account ID:
+GDGOKHIRX63EIAXKVI77BQV7LMDUH7DR4BMDDU77DJUXLPUU5HAXGN64
+Secret Seed:
+SD5ALH5UA7LRMN6ZB2QILW35YZTEOE37M3JVNAKZ3KWT7QSC77ABJ3ID
+```
+
+You will use your key pair to receive funds (account ID) and to sign your transaction (secret seed). You can share your account ID publicly but you must keep your seed secret. Otherwise, someone may steel your funds!
+
+To be able to submit transactions to Stellar network you must have an [account](https://www.stellar.org/developers/learn/concepts/accounts.html). Each account on Stellar network must have a [minimum balance](https://www.stellar.org/developers/learn/concepts/fees.html#minimum-balance) of 20 XLM ([lumens](https://www.stellar.org/developers/learn/concepts/assets.html#lumens-xlm-)).
+
+In test network you can use Friendbot to send you a small amount of lumens for testing purposes (remember to change `addr` param to your account ID`:
+```
+$ curl "https://horizon-testnet.stellar.org/friendbot?addr=GDGOKHIRX63EIAXKVI77BQV7LMDUH7DR4BMDDU77DJUXLPUU5HAXGN64"
+{
+  "_links": {
+    "transaction": {
+      "href": "https://horizon-testnet.stellar.org/transactions/f26916463a434c8b2ba28df657f3044d8295ddfa4b146a2f35788e7b98dd4f60"
+    }
+  },
+  "hash": "f26916463a434c8b2ba28df657f3044d8295ddfa4b146a2f35788e7b98dd4f60",
+  "ledger": 2436331,
+  "envelope_xdr": "AAAAAGXNhLrhGtltTwCpmqlarh7s1DB2hIkbP//jgzn4Fos/AAAAZAAAAA0AABP4AAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAzOUdEb+2RALqqj/wwr9bB0P8ceBYMdP/Gml1vpTpwXMAAAAXSHboAAAAAAAAAAAB+BaLPwAAAEDGbb/VRB0m2chzGFlKRABM7Uj+lWfMNpwxxwyUf8nsW3zRCZhjXdJKlmX3+Th7nvQyGE0dRMpzQt309vFNMoUH",
+  "result_xdr": "AAAAAAAAAGQAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAA=",
+  "result_meta_xdr": "AAAAAAAAAAEAAAACAAAAAAAlLOsAAAAAAAAAAMzlHRG/tkQC6qo/8MK/WwdD/HHgWDHT/xppdb6U6cFzAAAAF0h26AAAJSzrAAAAAAAAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAAAAAQAlLOsAAAAAAAAAAGXNhLrhGtltTwCpmqlarh7s1DB2hIkbP//jgzn4Fos/ACJ91JY4igAAAAANAAAT+AAAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAA"
+}
+```
+
+The JSON object you see is a response from [Horizon server](https://www.stellar.org/developers/horizon/learn/index.html) which acts as the interface between [stellar-core](https://github.com/stellar/stellar-core) and applications that want to access the Stellar network. You can learn more about the meaning of each of these fields in [Post Transaction](https://www.stellar.org/developers/horizon/reference/transactions-create.html) page. We will be using Horizon in the next section.
+
+Your account is now created and you can build and submit your first transaction.
+
+## Building transactions
+
+Let's check the code first and then we will explain it step by step.
+
+`transaction.js` file:
+```js
```

Comment:
I wonder if adding in comments inline with the code would be better. Also, this is very similar to the one in the js-stellar-sdk basic-examples page.